### ISSUE TITLE
when i importing screens from sketch

### ISSUE THREAD

**Issue by Omaretz** [avocode/avocode#1601]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.14.8`
Avocode UID: 170010
```
Error: Api#updateUploadSlot()() -> error (status: 400, error: {"project":["Slot target already set"]})
    at Object.normalizeError (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:16897)
    at Object.normalizeResponse (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:16733)
    at t.normalizeResponse (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2003680)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2000698
    at Request._callback (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2002521)
    at Request.self.callback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:186:22)
    at emitTwo (events.js:111:20)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:1081:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:1001:12)
    at Object.onceWrapper (events.js:290:19)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by jakubzitny** [comment]
Hi @Omaretz, thanks for reporting this. Can you please give us more details on how did this happen? Is there any chance you were syncing two different Sketch files, or one file multiple times? Does it happen to all uploads or only specific ones? Have you managed to upload anything to Avocode at all, please?

**Comment by bolekkerous** [comment]
Hey, @Omaretz -
could you please try to log out and log in again? It seemed to help in a similar issue here: #1611

**Comment by jakubzitny** [comment]
Hi there, this should be fixed in latest Avocode `2.15.0`. You can download it from [avocode.com/download](https://avocode.com/download) or use auto-update to install it. Please give it a try and let us know if you have any more issues.
